tsc-responsibilities.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: TSC responsibilities
+nav_order: 21
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# The Hyperledger Technical Steering Committee (TSC) responsibilities
+
+The [Hyperledger Charter](https://www.hyperledger.org/about/charter) states that the TSC is responsible for:
+
+1. coordinating the technical direction of the Hyperledger Foundation;
+
+2. approving project proposals (including, but not limited to, incubation, deprecation and changes to a project’s charter or scope) in accordance with a project lifecycle document to be developed, approved and maintained by the TSC;
+
+3. creating sub-committees or working groups to focus on cross-project technical issues or opportunities;
+
+4. communicating with external and industry organizations concerning related technical matters;
+
+5. appointing representatives to work with other open source or standards communities;
+
+6. establishing community norms, workflows, or policies for releases;
+
+7. discussing, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
+
+8. establishing election processes for Maintainers or other leadership roles in the technical community that are not within the scope of any single project.
+
```

Comment:
I know this is kind of implicitly expected but probably worth adding here:

9. Subscribe to the tsc@ mailing list to stay aware of the TSC related updates and issues.

Reply:
I can add that to the second list.